src/core/CLucene/search/query/DcoIdSetIterator.h

```diff
@@ -2,15 +2,11 @@
 
 #include "CLucene/index/DocRange.h"
 
+#include <string>
+
 class DocIdSetIterator {
```

Comment:
why delete all virtual functions for DocIdSetIterator, if so, what is the need of this class?

Reply:
delete DocIdSetIterator class